src/components/DescriptionCocktail.jsx

```diff
@@ -0,0 +1,84 @@
+import { useState, useEffect } from 'react';
+import { Box, UnorderedList, ListItem, Spacer, Image, Stack, StackDivider, Heading, Text, Wrap, WrapItem } from '@chakra-ui/react';
+import { useParams } from 'react-router-dom';
+
+const DescriptionCocktail = () => {
+  const [cocktail, setCocktail] = useState(null);
+  const params = useParams();
+  const { drinkId } = params();
```

Comment:
```suggestion
  const { drinkId } = params;
```
`params` n'est pas une fonction